properties/P000168.md

```diff
@@ -1,11 +1,25 @@
 ---
 uid: P000168
-name: $\omega$C
+name: Countable sets are discrete
 aliases:
 - Countable sets are closed
+- Cid
+- $\omega$C
 refs:
+- mathse: 4995501
+  name: Answer to "Does every door space with an infinite compact set have a single non-isolated point?"
+- zb: "0665.54003"
+  name: On spaces whose denumerable subspaces are discrete (Ganster, Reilly, Vamanamurthy)
 - mathse: 4751818
   name: Answer to What separation is required to ensure extremally disconnected spaces are sequentially discrete?
 ---
 
-Countable subsets are closed.
+Every {P57} subspace is {P52}.
+
+Equivalently (see comments of {{mathse:4995501}}), every {P57}
+subset is closed in the space.
+
+Defined as "cid" in {{zb:0665.54003}} (pg. 283 of 
+[https://gdz.sub.uni-goettingen.de/id/PPN311571026_0039](https://gdz.sub.uni-goettingen.de/id/PPN311571026_0039?tify=%7B%22pages%22%3A%5B299%5D%2C%22view%22%3A%22%22%7D))
+with the weakening that only non-{P78} {P57} subspaces must
+be {P52}.
```

Comment:
```suggestion
with the weaker meaning that "countable infinite subspaces are discrete".  If $X$ is infinite, the two notions are equivalent.
```
This better explains the abbreviation "cid" (and no need to use P57, P52 as links, as they already appear further up in the text).